common/integration_testing/src/google_smart_card_integration_testing/integration-test-controller.js

```diff
@@ -68,6 +84,9 @@ GSC.IntegrationTestController = class {
       this.executableModuleRequester_.dispose();
       this.executableModule.dispose();
       this.propertyReplacer.reset();
+
+      goog.testing.TestCase.getActiveTestCase().promiseTimeout =
```

Comment:
why do we need to reset it back? this value only affect the current test case, right?

Reply:
Done, you're right!